dotnet/DDB.Bindings/DroneDB.cs

```diff
@@ -34,10 +35,24 @@ static string GetLastError()
 
         public static string Init(string directory)
         {
-            if (_Init(directory, out var outPath) == DDBError.DDBERR_NONE)
-                return Marshal.PtrToStringAnsi(outPath);
+            if (string.IsNullOrWhiteSpace(directory))
```

Comment:
Should this logic be moved to the C API?

Reply:
See https://github.com/DroneDB/DroneDB/pull/115#issuecomment-709561637